cla_public/templates/checker/result/eligible.html

```diff
@@ -2,16 +2,13 @@
 
 {% block page_text %}
   <h1>
-    {%- if session.checker.need_more_info -%}
-      {{ _('We need more information') }}
-    {%- else -%}
-      {% trans %}Contact Civil Legal Advice{% endtrans %}
-    {%- endif -%}
+    {% trans %}Contact Civil Legal Advice{% endtrans %}
```

Comment:
I'd suggest using `{{ _('...') }}` for single lines. Maybe even more multiple lines.